recipe/meta.yaml

```diff
@@ -36,9 +36,9 @@ requirements:
 
 test:
   requires:
-    - r-testthat
+    - r-tinytest
   commands:
-    - Rscript -e 'testthat::test_package("tiledb")'
+    - Rscript -e 'tinytest::test_package("tiledb")'
```

Comment:
Good catch. We switched that before 0.8.0 when bulk testing as `tinytest` is much lighter on resource and starts faster (which mattered when we ran a lot of `valgrind`).

You could avoid the issue by using a more standard `R CMD check ...` where `...` could be the directory or, in the generally recommended form (as per the R manual), the `tar.gz` created by `R CMD build ...`.

Reply:
Seems conda-forge isn't packaging r-tinytest

Reply:
https://github.com/conda-forge/staged-recipes/pull/12881